src/components/QuranReader/SidebarNavigation/PageSelection.tsx

```diff
@@ -19,6 +22,7 @@ const PageSelection = () => {
       searchPlaceholder={t('sidebar.search-page')}
       renderItem={(page) => `${t('page')} ${page.label}`}
       isJuz={false}
+      selectedItem={selectedPage}
```

Comment:
nit: no need for `selectedPage` var since it's only used once. We can set `selectedItem={Number(lastReadPage)}` directly